### DISCUSSION TITLE
Downgrade supabase postgres 15 to version 14

### DISCUSSION THREAD

**Discussion by DrikaMakerdev** [supabase/supabase#19973] [Questions]
Hi guys, 

Due to a limitation of my application I need to use postgres version 14 in all my projects. How do I create my supabase projects with postgres version 14?

**Comment by GaryAustin1** [comment]
I can't find anything on this and know of no way to do that.  If pro you should contact support and see if they can help.  If free you could ask if there is a way, but not sure they support spinning up new free plans if each requires special work.
